examples/api/lib/material/popup_menu/popup_menu.2.dart

```diff
@@ -0,0 +1,131 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+
+/// Flutter code sample for [PopupMenuButton].
+
+// This is the type used by the popup menu below.
+enum SampleItem { preview, share, getLink, remove, download }
+
+void main() => runApp(const PopupMenuApp());
+
+class PopupMenuApp extends StatelessWidget {
+  const PopupMenuApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: PopupMenuExample(),
+    );
+  }
+}
+
+class PopupMenuExample extends StatefulWidget {
+  const PopupMenuExample({super.key});
+
+  @override
+  State<PopupMenuExample> createState() => _PopupMenuExampleState();
+}
+
+class _PopupMenuExampleState extends State<PopupMenuExample> {
+  SampleItem? selectedMenu;
+  bool customizedThemeAnimation = false;
+  bool slowAnimation = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.only(top: 50),
+          child: Align(
+            alignment: Alignment.topCenter,
+            child: Row(
+              mainAxisSize: MainAxisSize.min,
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                OutlinedButton(
+                  onPressed: () {
+                    setState(() {
+                      customizedThemeAnimation = !customizedThemeAnimation;
+                    });
+                  },
+                  child: Text(customizedThemeAnimation ? 'Remove Customization' : 'Customize Popup Animation'),
+                ),
+                const SizedBox(width: 10),
+                OutlinedButton(
+                  onPressed: () {
+                    setState(() {
+                      slowAnimation = !slowAnimation;
+                      if (!slowAnimation) {
+                        timeDilation = 1;
+                      } else {
+                        timeDilation = 5;
+                      }
+                    });
+                  },
+                  child: Text('${slowAnimation ? 'Slow' : 'Normal'} Animation'),
+                ),
+                PopupMenuButton<SampleItem>(
+                  popUpAnimationOveride: customizedThemeAnimation ? AnimationOverride(
+                    curve: Easing.emphasizedDecelerate,
+                    duration: Durations.extralong1,
+                  ) : null,
+                  icon: const Icon(Icons.more_vert),
+                  initialValue: selectedMenu,
+                  // Callback that sets the selected popup menu item.
```

Comment:
This comment isn't necessary. Even less so if the body of the setState is `selectedMenuItem = item` :-)